libs/stanag4609/src/main/java/org/codice/alliance/libs/stanag4609/AsynchronousMetadataPacket.java

```diff
@@ -19,6 +19,12 @@
 class AsynchronousMetadataPacket extends AbstractMetadataPacket {
     private static final int ASYNCHRONOUS_PES_PACKET_HEADER_LENGTH = 9;
 
+    /**
+     * Location of the "PES header length" field which contains the number of
+     * extra bytes contained in the header.
+     */
+    private static final int PES_HEADER_LENGTH_INDEX = 8;
+
     AsynchronousMetadataPacket(final byte[] pesPacketBytes, final PESPacket pesHeader,
```

Comment:
The original purpose of having AsynchronousMetadataPacket and SynchronousMetadataPacket was because SMPs had a one of the optional PES headers. But since the extra data was being skipped over, I can reduce AMP and SMP into a single MP class.